app/routes/moon_routes.py

```diff
@@ -0,0 +1,9 @@
+# from app import db
+# from app.models.planet import Planet
+# from app.models.moon import Moon
+# from flask import Blueprint, jsonify, abort, make_response, request
+# from .helper import validate_planet
+
+# moons_bp = Blueprint("moons", __name__, url_prefix="/moons")
+
+# do we even need this
```

Comment:
Nope, you don't need this file unless you're planning on adding future enhancements for moons 🌚 ✨